### PR TITLE
Use the summary details from food for the textile sidebar

### PR DESCRIPTION
**Pull request by magopian**
[user story](https://www.notion.so/fd0068a7db8348c584cd53a3eea399f2?v=37bf8aeb782747c3bf744ecaeafefd04&p=89655f01e0e042a7a5c2c3955b4b20e2&pm=s)

### CODE REVIEW

**Review by n1k0 — COMMENTED** [review]
This is a very good patch, congrats!

I'm still unsure we want to visually regroup 4 textile steps as a "transform" one, even if it makes sense. We'll have to check with POs.

Also, the Examples page should be updated to leverage tab switching and adjust the styles of the "Reprendre cette simulation" button.

image

File: src/Data/Textile/Simulator.elm [hunk 1]
```diff
@@ -153,6 +156,53 @@ compute db query =
...
+    , steps =
+        { materials = getImpacts Label.Material |> getImpact
+        , transform =
+            [ getImpacts Label.Spinning
+            , getImpacts Label.Fabric
+            , getImpacts Label.Ennobling
+            , getImpacts Label.Making
+            ]
+                |> Impact.sumImpacts
+                |> getImpact
```
[the remark this answers is not in the document]
Comment (magopian) [reply src/Data/Textile/Simulator.elm:196]: Yeah, it makes sense, and it's taken from the notion card, cf the "tableau de correspondance" written by Pascal.

File: src/Data/Food/Builder/Recipe.elm [hunk 2]
```diff
@@ -105,14 +107,26 @@ type alias Results =
...
-type alias Scoring =
-    { all : Unit.Impact
-    , allWithoutComplements : Unit.Impact
-    , complements : Unit.Impact
-    , climate : Unit.Impact
-    , biodiversity : Unit.Impact
-    , health : Unit.Impact
-    , resources : Unit.Impact
+resultsToImpactTabsConfig : Definition.Trigram -> Results -> ImpactTabs.Config
+resultsToImpactTabsConfig trigram results =
```
[the remark this answers is not in the document]
Comment (magopian) [reply src/Data/Food/Builder/Recipe.elm:111]: Does it make more sense to move it to `ImpactTabs`, or rather to some view related to food, like `Page.Food.Builder`?

[the remark this answers is not in the document]
Comment (n1k0) [reply src/Data/Food/Builder/Recipe.elm:111]: I'd go with the view module directly

**Review by n1k0 — APPROVED** [review]
LGTM, only remains the styling of the exemple box buttons on the Examples page which I can take care of in a next PR if you prefer.

**Comment by magopian** [comment]
Pushed a new commit with your feedback, thanks! Once it's green, I'll land if it's good for you @n1k0

**Comment by n1k0** [comment]
@magopian ship it! 🛳️

**Pull request merged**